### PR TITLE
Problem: app don't have access to current keyring-backend config

### PR DESCRIPTION
**Pull request by yihuang**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (evm) [#469](https://github.com/crypto-org-chain/ethermint/pull/469) More aggressive cache of common parameters in object store.
 * (evm) [#470](https://github.com/crypto-org-chain/ethermint/pull/470) State machine always use `ethtypes.MakeSigner` based on current chain config and block number, this is
   more accurate.
+* (app) [#]() Make keyring-backend client config accessible in app.
```
Comment (yihuang) [inline CHANGELOG.md]:
```suggestion
* (app) [#483](https://github.com/crypto-org-chain/ethermint/pull/483) Make keyring-backend client config accessible in app.
```

**Review by mmsqe — APPROVED** [review]

File: server/start.go [hunk 2]
```diff
@@ -149,14 +149,17 @@ which accepts a path for the resulting pprof file.
 			serverCtx.Logger.Info("Unlocking keyring")
 
 			// fire unlock precess for keyring
-			keyringBackend, _ := cmd.Flags().GetString(flags.FlagKeyringBackend)
```
Comment (mmsqe) [inline server/start.go:152]: seems we hasn't support keyring-backend flag in start cmd as ethermint

Comment (yihuang) [reply server/start.go:152]: good catch, added now

Comment (mmsqe) [reply server/start.go:152]: ethermint add [global](https://github.com/crypto-org-chain/ethermint/blob/develop/server/flags/flags.go#L95) but cronos doesn't yet

Comment (yihuang) [reply server/start.go:152]:
```
$ ./build/ethermintd start --help|grep keyring
      --keyring-backend string   Select keyring's backend (default "os")
```

hmm, it seems it's there already before, why you say that?

Comment (yihuang) [reply server/start.go:152]: > ethermint add [global](https://github.com/crypto-org-chain/ethermint/blob/develop/server/flags/flags.go#L95) but cronos doesn't yet

I see, cronos can call `AddGlobalFlags` as well then.

File: server/start.go [hunk 3]
```diff
@@ -225,6 +228,8 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().Uint64(server.FlagStateSyncSnapshotInterval, 0, "State sync snapshot interval")
 	cmd.Flags().Uint32(server.FlagStateSyncSnapshotKeepRecent, 2, "State sync snapshot to keep")
 
+	cmd.Flags().String(flags.FlagKeyringBackend, keyring.BackendOS, "Select keyring's backend (os|file|test)")
+
```
Comment (yihuang) [inline server/start.go]:
```suggestion
```

**Pull request merged**